website/docs/installation/docker-compose.md

```diff
@@ -49,55 +49,57 @@ AUTHENTIK_EMAIL__TIMEOUT=10
 
 ```shell
 AUTHENTIK_PORT_HTTP=80
 AUTHENTIK_PORT_HTTPS=443
 ```
 
-Afterwards, make sure to run `docker-compose up -d`.
+Be sure to run `docker-compose up -d` to rebuild with the new port numbers.
 
 ## Startup
 
-Afterwards, run these commands to finish
+Afterwards, run these commands to finish:
 
 ```shell
 docker-compose pull
 docker-compose up -d
 ```
 
-The compose file statically references the latest version available at the time of downloading the compose file, which can be overridden with the `AUTHENTIK_TAG` environment variable.
+The `docker-compose.yml` file statically references the latest version available at the time of downloading the compose file, which can be overridden with the `AUTHENTIK_TAG` environment variable.
 
-authentik will then be reachable on port 9000 (HTTP) and port 9443 (HTTPS).
+authentik is then reachable (by default) on port 9000 (HTTP) and port 9443 (HTTPS).
 
-To start the initial setup, navigate to `https://<your server>/if/flow/initial-setup/`. There you will be prompted to set a password for the akadmin user.
+To start the initial setup, navigate to `https://<your server>/if/flow/initial-setup/`. If you are using your local host, this will be `http://localhost:9000/if/flow/initial-setup/`.
+
+There you will be prompted to set a password for the akadmin user (the default user).
 
 ## Explanation
 
 :::warning
 The server assumes to have local timezone as UTC.
-All internals are handled in UTC, whenever a time is displayed to the user in UI it gets localized.
+All internals are handled in UTC; whenever a time is displayed to the user in UI it gets localized.
 Do not update or mount `/etc/timezone` or `/etc/localtime` in the authentik containers.
 This will not give any advantages.
 On the contrary, it will cause problems with OAuth and SAML authentication,
 e.g. [see this GitHub issue](https://github.com/goauthentik/authentik/issues/3005).
 :::
 
-The docker-compose project contains the following containers:
+The Docker-Compose project contains the following containers:
 
 -   server
 
-    This is the backend service, which does all the logic, runs the API and the actual SSO part. It also runs the frontend, hosts the JS/CSS files, and also serves the files you've uploaded for icons/etc.
+    This is the backend service, which does all the logic, plus runs the API and the SSO functionality. It also runs the frontend, hosts the JS/CSS files, and serves the files you've uploaded for icons/etc.
 
 -   worker
 
     This container executes background tasks, everything you can see on the _System Tasks_ page in the frontend.
 
--   redis & postgresql
+-   redis (for cache)
 
-    Cache and database respectively.
+-   postgresql (default database)
 
 Additionally, if you've enabled GeoIP, there is a container running that regularly updates the GeoIP database.
```

Comment:
```suggestion
```

This can actually be removed since GeoIP is built-in by default without an extra container, or we can maybe link https://goauthentik.io/docs/core/geoip#external-updates

Reply:
great, I'll add a link in separate PR.